### PR TITLE
Remove unneeded category in cd template

### PR DESCRIPTION
**Pull request by timja**
https://github.com/github/linguist/blob/master/lib/linguist/languages.yml

### CODE REVIEW

**Comment by timja** [comment]
@daniel-beck is this worth doing or better separated out?

**Comment by daniel-beck** [comment]
I'd not do this, that's the whole reason for https://github.com/jenkinsci/.github/pull/69

If linguist says it fits the repo's language, this gets grouped with all the default stuff. The current version keeps it separate in the "By Jenkins" group.

File: workflow-templates/cd.properties.json [hunk 1]
```diff
@@ -4,9 +4,9 @@
     "iconName": "jenkins",
     "categories": [
         "Deployment",
-        "Maven"
+        "Maven POM"
```
Comment (timja) [inline workflow-templates/cd.properties.json:7]:
```suggestion
```

File: workflow-templates/cd.properties.json [hunk 2]
```diff
@@ -4,9 +4,9 @@
     "iconName": "jenkins",
     "categories": [
         "Deployment",
```
Comment (timja) [inline workflow-templates/cd.properties.json:6]:
```suggestion
        "Deployment"
```

**Review by daniel-beck — APPROVED** [review]
Should have the expected result. Untested.

**Pull request merged**